src/test/java/ancaMarian/tests/tema5/Tema5Tests.java

```diff
@@ -0,0 +1,76 @@
+package ancaMarian.tests.tema5;
+
+import ancaMarian.driver.BrowserManager;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Tema5Tests {
+    public static void main(String[] args) {
+        ChromeDriver driver = BrowserManager.createDriver();
+        HerokuApp page = new HerokuApp("https://testpages.herokuapp.com/styled/index.html", driver);
+//primul Link din tema de pe platforma
+        page.checkFirstLink();
+        System.out.println(page.checkFirstLinkPageTitle());
+        System.out.println(page.checkFirstLinkPageExplanation());
+        System.out.println(page.checkFirstLinkFirstParagraph());
+        System.out.println(page.checkFirstLinkSecondParagraph());
+        page.returnToIndex();
+        page.checkSecondLink();
+        System.out.println(page.checkSecondLinkPageTitle());
+        System.out.println(page.checkSecondLinkPageExplanation());
+        System.out.println(page.checkSecondLinkFirstParagraphTitle());
+        System.out.println(page.checkSecondLinkFirstParagraph());
+        System.out.println(page.checkSecondLinkFirstParagraphAttributes());
+        System.out.println(page.checkSecondLinkSecondParagraphTitle());
+        System.out.println(page.checkSecondLinkSecondParagraph());
+        System.out.println(page.checkSecondLinkSecondParagraphAttributes());
+        page.secondLinkClickButton();
+        page.returnToIndex();
+        page.checkThirdLink();
+        System.out.println(page.checkThirdLinkPageTitle());
+        System.out.println(page.checkThirdLinkPageExplanation());
+        page.checkThirdLinkListOfParagraphs();
+        page.checkThirdLinkListOfLinks();
+        page.returnToIndex();
+        page.checkFourthLink();
+        System.out.println(page.checkFourthLinkPageTitle());
+        System.out.println(page.checkFourthLinkPageExplanation());
+        System.out.println(page.checkFourthLinkTable());
+
```

Comment:
No need to add 2 blank lines. Please remove one.

Reply:
done